substrate/frame/membership/src/migrations/v4.rs

```diff
@@ -47,7 +47,7 @@ pub fn migrate<T: frame_system::Config, P: GetStorageVersion + PalletInfoAccess,
 			target: LOG_TARGET,
 			"New pallet name is equal to the old prefix. No migration needs to be done.",
 		);
-		return Weight::zero()
+		return Weight::zero();
```

Comment:
Please revert unrelated changes.

Reply:
Oh, my bad, this was done by `cargo fmt` 😅